src/rubicon/objc/api.py

```diff
@@ -214,6 +252,17 @@ def __call__(self, receiver, *args, convert_args=True, convert_result=True):
         else:
             converted_args = args
 
+        # Init methods consume their `self` argument (the receiver), see
+        # https://clang.llvm.org/docs/AutomaticReferenceCounting.html#semantics-of-init.
+        # To avoid segfaults on garbage collection if `init` does not return `self` but
```

Comment:
This affects not just garbage collection, but any subsequent use of the object.

```suggestion
        # To avoid segfaults on garbage collection if `init` does not return `self` but
        # To ensure the receiver pointer remains valid if `init` does not return `self` but
```

Reply:
Updated now, thanks!